pl_bolts/models/detection/yolo/yolo_layers.py

```diff
@@ -0,0 +1,487 @@
+        significantly (IoU greater than ``self.ignore_threshold``).
+
+        Args:
+            boxes: The predicted corner coordinates in the image space. Tensor of size
+                ``[batch_size, height, width, boxes_per_cell, 4]``.
+            targets: List of dictionaries of ground-truth targets, one dictionary per image.
+
+        Returns:
+            A boolean tensor shaped ``[batch_size, height, width, boxes_per_cell]`` with ``False``
+            where the predicted box overlaps a target significantly and ``True`` elsewhere.
+        """
+        batch_size, height, width, boxes_per_cell, num_coords = boxes.shape
+        num_preds = height * width * boxes_per_cell
+        boxes = boxes.view(batch_size, num_preds, num_coords)
+
+        results = torch.ones((batch_size, num_preds), dtype=torch.bool, device=boxes.device)
+        for image_idx, (image_boxes, image_targets) in enumerate(zip(boxes, targets)):
+            target_boxes = image_targets['boxes']
+            if target_boxes.shape[0] > 0:
+                ious = box_iou(image_boxes, target_boxes)  # [num_preds, num_targets]
+                best_iou = ious.max(-1).values  # [num_preds]
+                results[image_idx] = best_iou <= self.ignore_threshold
+
+        return results.view((batch_size, height, width, boxes_per_cell))
+
+    def _calculate_losses(
+        self, boxes: Tensor, confidence: Tensor, classprob: Tensor, targets: List[Dict[str, Tensor]],
+        image_size: Tensor, lc_mask: Tensor
+    ) -> Dict[str, Tensor]:
+        """
+        From the targets that are in the image space calculates the actual targets for the network
+        predictions, and returns a dictionary of training losses.
+
+        Args:
+            boxes: The predicted bounding boxes. A tensor sized
+                ``[batch_size, height, width, boxes_per_cell, 4]``.
+            confidence: The confidence predictions, normalized to `[0, 1]`. A tensor sized
+                ``[batch_size, height, width, boxes_per_cell]``.
+            classprob: The class probability predictions, normalized to `[0, 1]`. A tensor sized
+                ``[batch_size, height, width, boxes_per_cell, num_classes]``.
+            targets: List of dictionaries of target values, one dictionary for each image.
+            image_size: Width and height in a vector that defines the scale of the target coordinates.
+            lc_mask: A boolean mask containing ``True`` where the predicted box does not overlap
+                any target significantly.
+
+        Returns:
+            A dictionary of training losses.
+        """
+        batch_size, height, width, boxes_per_cell, _ = boxes.shape
+        device = boxes.device
+        assert batch_size == len(targets)
+
+        # A multiplier for scaling image coordinates to feature map coordinates
+        grid_size = torch.tensor([width, height], device=device)
+        image_to_grid = grid_size / image_size
```

Comment:
```suggestion
        image_to_grid = torch.true_divide(grid_size, image_size)
```
Could you use `torch.true_divide()` instead of `/` here, too?

Reply:
Yes, that seems to be the problem. Fixed.